iib/workers/tasks/build.py

```diff
@@ -227,12 +234,12 @@ def _get_image_arches(pull_spec):
     return arches
 
 
-def get_rebuilt_index_image(request_id):
+def get_rebuilt_image_pull_spec(request_id):
     """
-    Generate the pull specification of the index image rebuilt by IIB.
+    Generate the pull specification of the image rebuilt by IIB.
```

Comment:
How about keeping it consistent throughout and having `container image` everywhere?

Reply:
I added as a separate commit because I had to touch other methods in this file.